src/Serokell/Data/Variant/Helpers.hs

```diff
@@ -18,4 +21,4 @@ none = VarNone
 -- | Create VarMap from Foldable containing pairs of Variants.
 -- TODO: maybe use better approach like `a := b` to create KeyValuePair.
 varMap :: Foldable t => t (Variant, Variant) -> Variant
-varMap = VarMap . HM.fromList . toList
+varMap = VarMap . HM.fromList . F.toList
```

Comment:
I think it's better to change type of `varMap` from `Foldable` to `Container` (to make it more compatible with `universum`) and use `toList` from `universum`